### ISSUE TITLE
Remove CPs to Documentation Folder

### ISSUE THREAD

**Issue by abouffard** [Esri/joint-military-symbology-xml#222]
Updated Issue:
I'm removing CPs from the repository.

Original Issue:
DISA has provided us with the complete list of approved 2525D Change 1 change proposals, available to-date (approved as of SSMC/JSP 15-2, July 2015).

They are now available [here](https://esri.app.box.com/files/0/f/4597672254/Approved_Change_Proposals) along with the voting record.

This task is to add those CPs that are not already present to our tracking database [here](https://github.com/Esri/joint-military-symbology-xml/tree/master/documentation/change_proposals) and add a new column to the readme to track implementation merge number, so progress in implementing them in JMSML can be tracked on a CP by CP basis.

As part of resolving this issue, we should also create individual issues for each tracked CP, so we have a place to log any discussions regarding each CP.

**Comment by csmoore** [comment]
Not sure if it is an unrelated question, but do all of the approved CPs that affect this repo get entered as separate issues here (I only checked for the first one 15-040 and couldn't find - but maybe you add issues as you add them to that table).

Also, while you are making this change could that tracking table format be changed so it is sorted by CP number (it is somewhat difficult to visually find a CP now)

**Comment by abouffard** [comment]
That was my intent.  I have updated the issue above to note that we should also create issues for the tracked CPs.  The CPs currently logged as issues are just the CPs we started tracking from SSMC 15-1 and I think these are all just ones we authored.

Yes, we can certainly re-order the CPs in the table by DISA CP reference number, which is different from our own reference number.

Most of the new CPs we will be adding will not have been authored by Esri and therefore won't have Esri reference numbers, so the DISA reference number column will become the new first column in that table.
